cmake/modules/FindPULSEAUDIO.cmake

```diff
@@ -1,3 +1,8 @@
+# SPDX-FileCopyrightText: 2008 Matthias Kretz <kretz@kde.org>
+# SPDX-FileCopyrightText: 2009 Marcus Hufgard <Marcus.Hufgard@hufgard.de>
+#
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Please paste the full text of the license in the file. I don't think it's sufficient to just name the SPDX identifier; the license needs to be distributed together with the code it applies to.

Reply:
Ok now it should be there